kart/sqlalchemy/adapter/postgis.py

```diff
@@ -71,6 +71,17 @@ class KartAdapter_Postgis(BaseKartAdapter, Db_Postgis):
 
     ZM_FLAG_TO_STRING = {0: "", 1: "M", 2: "Z", 3: "ZM"}
 
+    @classmethod
+    def v2_column_schema_to_sql_spec(cls, col, v2_obj=None, has_int_pk=False):
+        col_name = cls.quote(col.name)
+        sql_type = cls.v2_type_to_sql_type(col, v2_obj)
+
+        # Make int PKs auto-increment.
+        if has_int_pk and col.pk_index is not None:
+            sql_type = sql_type.replace("INT", "SERIAL")
```

Comment:
there's also `SMALLSERIAL` (16bit) and `BIGSERIAL` (64bit)

Reply:
oh... this replaces the substring in the type name 😬  Comment?

Reply:
Added a comment.
Forgot about INTEGER - fixed this bug.